getting-started/glossary.md

```diff
@@ -0,0 +1,48 @@
+# GLOSSARY   
+
+## API
+Application Programming Interface. A software intermediary that allows two applications to talk to each other.
+
+
+## API Endpoint
+The place that APIs send requests to and where the resource resides.
+
+## API Gateway
+An API management tool that sits between a client and a collection of backend services.
+
+## API Gateway Id
+
+
+## API Key
+A unique identifier used by CFT to authenticate a calling program, this is used for rate limiting. It is assigned to an individual project.
+
+## API Request
+A call to a server using an API.
+
+## API Response
+Response sent by the server according to the input header of the API request.
+
+## Client Id and Secret
+
+A public identifier for applications. Required to generate an authorization token to make API calls.
```

Comment:
this is considered your credentials. which you shouldn't make it public. can we not use "public" as this can be confusing.